src/sage/rings/padics/FM_template.pxi

```diff
@@ -869,7 +869,7 @@ cdef class FMElement(pAdicTemplateElement):
 
 cdef class pAdicCoercion_ZZ_FM(RingHomomorphism):
     """
-    The canonical inclusion from ZZ to a fixed modulus ring.
+    The canonical inclusion from ``ZZ`` to a fixed modulus ring.
 
```

Comment:
single-quoted `\ZZ`?

Reply:
here and below. But you seem to have intended ``ZZ``.

Reply:
I've standardized on single-quoted `\ZZ` now